mon-pix/tests/acceptance/challenge-qcu-test.js

```diff
@@ -70,16 +45,115 @@ describe('Acceptance | Displaying a QCU', function() {
       expect(find('.alert').textContent.trim()).to.equal('Pour valider, sélectionner une réponse. Sinon, passer.');
     });
 
-    it('should only display an error alert if the user tries to validate after having interacting once with the page', async function() {
+    it('should hide the alert error after the user interact with radio button', async () => {
       // given
-      findAll('input[type=radio][name="radio"]')[1].checked = false;
       await click('.challenge-actions__action-validate');
 
       // when
-      await click(findAll('.label-checkbox-proposal')[0]);
+      await click(findAll('.proposal-text')[1]);
 
       // then
       expect(find('.alert')).to.not.exist;
     });
+
+    it('should go to checkpoint when user validated', async () => {
+      // when
+      await click(findAll('.proposal-text')[1]);
+      await click('.challenge-actions__action-validate');
+
+      // then
+      expect(currentURL()).to.contains(`/assessments/${assessment.id}/checkpoint`);
+    });
+
+  });
+
+  describe('When challenge is already answered', () => {
+    beforeEach(async () => {
+      // given
+      server.create('answer', {
+        value: '2',
+        result: 'ko',
+        assessment,
+        challenge: qcuChallenge,
+      });
+
+      // when
+      await visit(`/assessments/${assessment.id}/challenges/${qcuChallenge.id}`);
+    });
+
+    it('should mark radio button corresponding to the answer and propose to continue', async () => {
+      // then
+      expect(findAll('input[type=radio][name="radio"]')[0].checked).to.be.false;
+      expect(findAll('input[type=radio][name="radio"]')[1].checked).to.be.true;
+      expect(findAll('input[type=radio][name="radio"]')[2].checked).to.be.false;
+      expect(findAll('input[type=radio][name="radio"]')[3].checked).to.be.false;
```

Comment:
Ne serait-ce pas mieux de sortir le `findAll` ?
```suggestion
const radioButtons = findAll('input[type=radio][name="radio"]';
expect(radioButtons[0].checked).to.be.false;
expect(radioButtons[1].checked).to.be.true;
expect(radioButtons[2].checked).to.be.false;
expect(radioButtons[3].checked).to.be.false;
```